### DISCUSSION TITLE
reduce accidental mode shifts (eg timelapse instead of video)

### DISCUSSION THREAD

**Discussion by tedder** [gopro/labs#346] [Feature Requests]
It'd be nice to have a feature that would force a mode (eg video only), or at least reduce the instances of "oops I accidentally changed modes". In other words, the power button would only turn power on/off, not change modes.

I looked through the feat requests and didn't see anything, but I know I'm not the only one who does this.

**Comment by dnewman-gpsw** [comment]
This sounds very like Archive mode, you get only only mode in this configuration. https://gopro.github.io/labs/control/archive/

**Reply by tedder** [reply]
I hadn't thought about using that! I'll think on it and try it out.
